brave/brave-core
# Removes link for add funds for bap

Resolves https://github.com/brave/brave-browser/issues/6874

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- clean profile
- enable rewards
- switch to Japan
- click on panel an tip some publisher
- make sure that you don't see add funds

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
